README.md

```diff
@@ -6,11 +6,263 @@
+
+It is recommended that you use the `.test` top level domain (TLD) for your local environments, as this TLD is reserved
+for the testing of software and is not intended to ever be installed into the global Domain Name System. Additionally,
+WP Local Docker is configured to send any container to container traffic for .test TLDs directly to the gateway
+container, so that things like WP Cron and the REST API can work between environments out of the box.
+
+### Migrate a WP Local Docker V1 Environment
+
+`10updocker migrate <OLD_PATH> [NEW_ENV]` will migrate an old standalone WP Local Docker environment into a new WP Local Docker V2
+environment. Before running this command, create a new environment using the `10updocker create` command.
+
+`OLD_PATH` should be the path to the root of your old WP Local Docker environment.
+
+`NEW_ENV` should specify what environment to import into. If omitted, you will be prompted to select from available environments
+
+Example:
+* `10updocker migrate ~/sites/mysite`
+
+### Delete an Environment
+
+`10updocker delete <hostname>` will delete an environment with the given hostname. Any local files, docker volumes, and
+databases related to the environment will be deleted permanently.
+
+A special hostname `all` is available that will delete all environments. You will be asked to confirm deletion of each
+environment.
+
+### Stop an Environment
+
+`10updocker stop <hostname>` will stop an environment from running while retaining all files, docker volumes, and
+databases related to the environment.
+
+A special hostname `all` is available that will stop all running environments as well as the global services.
+
+### Start an Environment
+
+`10updocker start <hostname>` will start a preexisting environment.
+
+A special hostname `all` is available that will start all environments as well as the global services.
+
+### Restart an Environment
+
+`10updocker restart <hostname>` will restart all services associated with a preexisting environment.
+
+A special hostname `all` is available that will restart all environments as well as the global services.
+
+### Upgrade an Environment
 
-## Like what you see?
+`10updocker upgrade <hostname>` will upgrade all services associated with a preexisting environment.
 
+This command will assist you with keeping your environments up to date with the most recent upstream changes. If you
+are running an environment that was created before `v2.6.0`, we recommend upgrading your environment for a noticeable
+performance increase.
+
+### Elasticsearch
+
+If you have enabled Elasticsearch for a particular environment, you can send requests from the host machine to the
+Elasticsearch server by prefixing the url path with `/__elasticsearch/`. For example, if you wanted to hit the
+`/_all/_search/` endpoint of Elasticsearch, the URL would look like: `http://<hostname>/__elasticsearch/_all/_search`
+
+### WP Snapshots
+
+See the section on [using WP Snapshots](../wp-snapshots)
```

Comment:
@tylercherpak 

This needs to be updated or add the section from [here](https://10up.github.io/wp-local-docker-docs/wp-snapshots/).